dyno-contrib/src/main/java/com/netflix/dyno/contrib/EstimatedHistogramBasedCounter.java

```diff
@@ -83,13 +83,19 @@ public Number getValue() {
 			return estHistogram.mean();
 		}
 
+        @Override
+        public Number getValue(int pollerIndex) {
```

Comment:
It is something that should just be an implementation detail of servo. This could get refactored more, but most of the changes I would prefer would break compatibility of the metrics produced. I'm not sure how of a concern that is or how much use this data gets used internally.

This should be fine for now and we can explore it further if there is enough demand.